wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -1194,11 +1198,118 @@ class PanelNode(BlockInclusionNode):
 register.tag("panel", PanelNode.handle)
 
 
-class FieldNode(BlockInclusionNode):
+class RawFormattedFieldNode(BlockInclusionNode):
+    content_var = "rendered_field"
     template = "wagtailadmin/shared/field.html"
 
 
-register.tag("field", FieldNode.handle)
+register.tag("rawformattedfield", RawFormattedFieldNode.handle)
+
+
+@register.inclusion_tag("wagtailadmin/shared/formatted_field.html")
+def formattedfield(
```

Comment:
Same comment as above re readability of the tag name. It’d also be nice if tag name and template name used the same case where possible.